MIGRATION.md

```diff
@@ -6,10 +6,12 @@
   - See `ios_generate_strings_file_from_code`, `ios_extract_keys_from_strings_files`, `ios_download_strings_files_from_glotpress` and `ios_merge_strings_files` for typical replacements.
 - The action `ios_get_app_version` now requires a parameter `public_version_xcconfig_file` with the public `.xcconfig` file path instead of relying on the environment variable `PUBLIC_CONFIG_FILE`. While the complete removal of this environment variable is our goal, at this point it is still required by other actions such as `ios_bump_version_release` and `ios_codefreeze_prechecks`.
 - The usage of a `Deliverfile` (including its `app_version`) is discouraged -- please use `upload_to_app_store` directly from your `Fastfile` instead. Therefore, the parameter `skip_deliver` from the actions `ios_bump_version_hotfix` and `ios_bump_version_release` has been removed.
+- If you were using the `get_prs_list` action, you should consider using `get_prs_between_tags` instead (which relies on the GitHub API and a config file, rather than milestones)
```

Comment:
From what I can see, `get_prs_list` was actually removed so there's not really an option to continue using it?

```suggestion
- If you were using the `get_prs_list` action, you should use `get_prs_between_tags` instead (which relies on the GitHub API and a config file, rather than milestones)
```

Reply:
To me `use` suggests it's the only solution and also that it would work the same way (as in just switch one for the other). While they are in fact different, as `get_prs_list` was based on the list of PRs in a given milestone, not the PRs that have been merged between two tags.

While by saying `consider using` here I meant "either you still want a list of PRs and you might be interested in `get_prs_between_tags`… _but_ if you go that route you'll have to change the process used by your app's release process accordingly, to rely on tags not on milestones for that list… _or_ maybe this use of `get_prs_list` in your client app was just legacy and in fact you haven't been using its result anymore and could be a good occasion to drop it completely"

I considered just renaming `get_prs_list` you something like `get_prs_in_milestone` instead and keeping it. But also, to my knowledge, we've stopped using that action a while ago in our client repos, replacing it with a manually-maintained `RELEASE-NOTES.txt` (that allows developers to provide better context for when those are intended to be used as a base for the final user-facing release notes). 

So I'm pretty sure that no repo use it anymore, or if their Fastfile calls it the file it generates isn't used by anyone anymore, so they might as well remove their call to it. And then, maybe consider if `get_prs_between_tags` could be useful to them… but at potentially a completely different step in the process (like when creating GitHub Pre-Releases for beta builds that happen in the middle of sprints/milestones.

Reply:
> To me use suggests it's the only solution and also that it would work the same way (as in just switch one for the other)

Indeed, to me that that would imply that the actions were interchangeable -- I didn't realise at first they weren't, thanks for the context.

> So I'm pretty sure that no repo use it anymore, or if their Fastfile calls it the file it generates isn't used by anyone anymore, so they might as well remove their call to it. And then, maybe consider if get_prs_between_tags could be useful to them… but at potentially a completely different step in the process (like when creating GitHub Pre-Releases for beta builds that happen in the middle of sprints/milestones.

If teams also started using a manually maintained `RELEASE-NOTES.txt`, what is the use case for the new action and who's going to use it? Perhaps this is what I'm missing from the `MIGRATION.md` note 🤔: if I use an older version, why should I change / adopt the new action (and what to do if I use the old one or not)?

Reply:
Yeah good point. Short answer is that this `get_prs_between_tags` is more to generate internal release notes for App Center builds and GitHub Pre-Releases (while `get_prs_list` was initially aimed to generate the `RELEASE-NOTES.txt` during a whole sprint to hand over to our freelance writer). But I agree that this might not be clear in this MIGRATION.md paragraph; I'll try to reword this.

Reply:
@iangmaia I tried to improve the wording in 536698540876c4877bef14b4ac7f2040d07ed828. Let me know what you think.

Reply:
Yes, it looks much clearer now 👍